### PR TITLE
Implement copy edits to templates

### PR DESCRIPTION
**Pull request by dwicinas**
Betsy edited the EPAS Red Hat family topics, and these changes should implement those edits into the install templates.

Here is the PR for Betsy's original edits: https://github.com/EnterpriseDB/docs/pull/4648

This PR implements the same sort of edits across install templates for all products.

### CODE REVIEW

**Comment by dwicinas** [comment]
@drothery-edb This is ready for review and possible merging. EPAS has some pretty substantial changes, but for all the other products, the changes are very simple--mostly colons and periods.

File: install_template/templates/products/edb-postgres-advanced-server/base.njk [hunk 1]
```diff
@@ -53,17 +53,17 @@ The server runs with the `peer` or `ident` permission by default. You can change
...
 ALTER ROLE enterprisedb IDENTIFIED BY password;
 ```
 
 ## Experiment
 
-Now you are ready to create and connect to a database, create a table, insert data in a table, and view the data from the table. 
+Now you're ready to create and connect to a database, create a table, insert data in a table, and view the data from the table. 
 
-First let's create a database to hold human resource information named `hr` using psql.
+First, use psql create a database named `hr` to hold human resource information.
```
Comment (drothery-edb) [inline install_template/templates/products/edb-postgres-advanced-server/base.njk:66]:
```suggestion
First, use psql to create a database named `hr` to hold human resource information.
```

**Review by drothery-edb — DISMISSED** [review]

File: install_template/templates/products/edb-postgres-advanced-server/base.njk (hunk 1 above)
Comment (dwicinas) [reply install_template/templates/products/edb-postgres-advanced-server/base.njk]: Fixed

**Comment by drothery-edb** [comment]
@dwicinas can you regenerate the files to get the small fix in them?

**Comment by dwicinas** [comment]
@drothery-edb I regenerated the files. Sorry I didn't think of that yesterday.

**Review by drothery-edb — APPROVED** [review]

**Pull request merged**